src/color.h

```diff
@@ -89,6 +91,16 @@ Color& Color::operator/=(float a)
 }
 
 
+Color& Color::gamma()
```

Comment:
Very stupid gamma function to liearize colors.

Reply:
emoji even here...